free-3.md

```diff
@@ -57,12 +65,18 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+If someone were to work on this project, each command they created would require quite a bit of programming in the host language to ensure that it worked correctly. Since there are already quite a few tools out there, it would also make the most sense for them to spend time looking at existing tools and then determining how to best subvert them to use for C++, so more of their time would probably be spent on things that are not directly part of the language aspects of this project.
 
```

Comment:
Definitely--no need to reinvent the wheel. Figuring out how other people have approached this would save a lot of time on your part.